oluwajuwon/Fast-Food-Fast
# Implement feedback on get all orders and updating orders

### What does this pull request do?
This pull request updates get all orders and updating an order routes 

### Description of  the Task to be completed?
These features ensures that when the admin user gets all orders, it also displays the name of the food that is being ordered and also while updating the order status, the admin can only update using a set of predefined words (New, Processing, Cancelled and Completed). The admin has to input the order status for the feature to work

### How should this be manually tested?
Users can test it by:
- Downloading the app Postman from - https://www.getpostman.com/
- Open up and postman and enter the URLs below:
  -  https://fastfood-fast-app.herokuapp.com/api/v1/orders - using the GET request (select the request 
      by the left side of the URL bar) to retrieve all orders and check for the food name
  -  https://fastfood-fast-app.herokuapp.com/api/v1/orders/<orderId> - using the PUT request to 
      update the order status of the specific order to check if the order status can be updated and to 
      check if it can be updated with only: New, Processing, Cancelled and Completed

### What are the relevant pivotal tracker stories?
Admin should be able to get orders and update the order - https://www.pivotaltracker.com/story/show/160875501
